### ISSUE TITLE
[BUG] Mint 21 - ORCA screen reader not working

### ISSUE THREAD

**Issue by christophe-g** [linuxmint/cinnamon#11146]
```
System:
  Kernel: 5.15.0-46-generic x86_64 bits: 64 compiler: gcc v: 11.2.0 Desktop: Cinnamon 5.4.11
    tk: GTK 3.24.33 wm: Mutter dm: LightDM Distro: Linux Mint 21 Vanessa base: Ubuntu 22.04 jammy
Machine:
  Type: Laptop System: Timi product: TM1613 v: A05 serial: <superuser required>
  Mobo: Timi model: TM1613 v: A05 serial: <superuser required> UEFI: Insyde v: A05
    date: 08/11/2016
```

### Issue
Turning LinuxMint 21 default screen reader on (either from `system settings/accessibility/screen reader` or from `terminal > orca` ) does not work as expected. 

Instead of interpreting the UI and user gestures, it reads a dummy sound. 

### Steps to reproduce
Turn screen reader on

### Expected behaviour
Obviously, have the screen reader work again, as it was under Mint 20.

### Other information

Log from speech dispatcher

```
[Mon Sep  5 20:39:18 2022 : 683532] speechd:   Speech Dispatcher Logging to file /run/user/1000/speech-dispatcher/log//speech-dispatcher.log
[Mon Sep  5 20:39:18 2022 : 683875] speechd:  Configuration has been read from "/etc/speech-dispatcher/speechd.conf"
[Mon Sep  5 20:39:18 2022 : 683947] speechd:   couldn't open directory /home/christophe/.local/share/../libexec/speech-dispatcher-modules because of error No such file or directory

[Mon Sep  5 20:39:18 2022 : 696406] speechd:   couldn't open directory /usr/lib/x86_64-linux-gnu/speech-dispatcher-modules because of error No such file or directory

[Mon Sep  5 20:39:18 2022 : 696442] speechd:  Initializing output module dummy with binary /usr/lib/speech-dispatcher-modules/sd_dummy and configuration /etc/speech-dispatcher/modules/dummy.conf
[Mon Sep  5 20:39:18 2022 : 696448] speechd:   Output module is logging to file /run/user/1000/speech-dispatcher/log//dummy.log
[Mon Sep  5 20:39:18 2022 : 696798] speechd:  Module dummy loaded.
[Mon Sep  5 20:39:18 2022 : 698236] speechd:  Module dummy started successfully with message: 
---------------
Everything ok so far.
---------------

[Mon Sep  5 20:39:18 2022 : 699280] speechd:  Error: Module reported error in request from speechd (code 3xx): 300-Opening sound device failed. Reason: Cannot open plugin server. error: file not found. 
300 MODULE ERROR
.
[Mon Sep  5 20:39:18 2022 : 707191] speechd:   Speech Dispatcher Logging to file /run/user/1000/speech-dispatcher/log//speech-dispatcher.log
[Mon Sep  5 20:39:18 2022 : 707210] speechd:   Speech Dispatcher started with 1 output module
[Mon Sep  5 20:39:18 2022 : 708149] speechd: Speech Dispatcher started and waiting for clients ...
[Mon Sep  5 20:39:19 2022 : 378033] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:19 2022 : 378052] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:19 2022 : 379095] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en-us/gender-neutral.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 379301] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en/gender-neutral.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 379330] speechd: Failed to load symbols 'gender-neutral.dic' for locale 'en-us'
[Mon Sep  5 20:39:19 2022 : 379383] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en-us/font-variants.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 379427] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en/font-variants.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 379443] speechd: Failed to load symbols 'font-variants.dic' for locale 'en-us'
[Mon Sep  5 20:39:19 2022 : 379509] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en-us/symbols.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 380129] speechd: Loading NUL byte entry is not yet supported
[Mon Sep  5 20:39:19 2022 : 380142] speechd: Invalid line in file /usr/share/speech-dispatcher/locale/en/symbols.dic: \0	blank	char	# null
[Mon Sep  5 20:39:19 2022 : 381324] speechd: Loading NUL byte entry is not yet supported
[Mon Sep  5 20:39:19 2022 : 381335] speechd: Invalid line in file /usr/share/speech-dispatcher/locale/base/symbols.dic: \0	blank	char	# null
[Mon Sep  5 20:39:19 2022 : 382436] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en-us/emojis.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 388107] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/base/emojis.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 396424] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en-us/orca.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 396462] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en/orca.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 396473] speechd: Failed to load symbols 'orca.dic' for locale 'en-us'
[Mon Sep  5 20:39:19 2022 : 396490] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en-us/orca-chars.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 396518] speechd: Failed to open file '/usr/share/speech-dispatcher/locale/en/orca-chars.dic': No such file or directory
[Mon Sep  5 20:39:19 2022 : 396527] speechd: Failed to load symbols 'orca-chars.dic' for locale 'en-us'
[Mon Sep  5 20:39:26 2022 : 720838] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:26 2022 : 720856] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:31 2022 : 465713] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:31 2022 : 465727] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:35 2022 : 328984] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:35 2022 : 329001] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:44 2022 : 987137] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:44 2022 : 987150] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:46 2022 : 40476] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:46 2022 : 40506] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:53 2022 : 34975] speechd:   Warning: Didn't find preferred output module, using default
[Mon Sep  5 20:39:53 2022 : 34989] speechd:   Couldn't load default output module, trying other modules
[Mon Sep  5 20:39:58 2022 : 494931] speechd: Terminating...
[Mon Sep  5 20:39:58 2022 : 494967] speechd:  Closing open connections...
[Mon Sep  5 20:39:58 2022 : 495683] speechd:  Closing open output modules...
[Mon Sep  5 20:39:58 2022 : 495710] speechd:   Unloading module name=dummy
[Mon Sep  5 20:39:58 2022 : 495728] speechd:   Closing module "dummy"...
[Mon Sep  5 20:39:58 2022 : 499125] speechd: ERROR: waitpid() failed when waiting for child (module).
[Mon Sep  5 20:39:58 2022 : 499197] speechd:  Closing server connection...
[Mon Sep  5 20:39:58 2022 : 499272] speechd:  Speech Dispatcher terminated correctly
[Mon Sep  5 20:40:02 2022 : 753898] speechd:   Speech Dispatcher Logging to file /run/user/1000/speech-dispatcher/log//speech-dispatcher.log
[Mon Sep  5 20:40:02 2022 : 754010] speechd:  Configuration has been read from "/etc/speech-dispatcher/speechd.conf"
[Mon Sep  5 20:40:02 2022 : 754069] speechd:   couldn't open directory /home/christophe/.local/share/../libexec/speech-dispatcher-modules because of error No such file or directory

[Mon Sep  5 20:40:02 2022 : 765169] speechd:   couldn't open directory /usr/lib/x86_64-linux-gnu/speech-dispatcher-modules because of error No such file or directory

[Mon Sep  5 20:40:02 2022 : 765238] speechd:  Initializing output module dummy with binary /usr/lib/speech-dispatcher-modules/sd_dummy and configuration /etc/speech-dispatcher/modules/dummy.conf
[Mon Sep  5 20:40:02 2022 : 765244] speechd:   Output module is logging to file /run/user/1000/speech-dispatcher/log//dummy.log
[Mon Sep  5 20:40:02 2022 : 765668] speechd:  Module dummy loaded.
[Mon Sep  5 20:40:02 2022 : 766616] speechd:  Module dummy started successfully with message: 
---------------
Everything ok so far.
---------------


```

**Comment by christophe-g** [comment]
Has anyone been able to reproduce this ? 
Thanks
C.

**Comment by christophe-g** [comment]
Same behavior for me after upgrading to Mint 21.1 Cinnamon. 

I would love to have someone either confirm the non-working behavior, or tell that it is working on their system.

**Comment by christophe-g** [comment]
In case someone encounters the same behavior: install  `espeakup` and `speech-dispatcher-espeak` from the software manager.
